### DISCUSSION TITLE
hi, i need your help, how to create a branch in an empty repository

### DISCUSSION THREAD

**Discussion by KirillDolbnya** [community/community#114257] [New to GitHub] [answered]
![image](https://github.com/community/community/assets/119807776/5d62fb02-6685-45cd-bd90-94e5d2a5be2d)
![image](https://github.com/community/community/assets/119807776/23ce98dc-6381-4af7-aa80-57e457500c8e)
![image](https://github.com/community/community/assets/119807776/5c46db21-e325-4c26-89a0-3a05137c33d7)

**Comment by KirillDolbnya** [comment]
I also created a riposte with a readme file, but the task did not count for me

**Reply by queenofcorgis** [reply]
@KirillDolbnya has the task still not counted for you?

**Reply by KirillDolbnya** [reply]
Yes @queenofcorgis

**Reply by queenofcorgis** [reply]
@KirillDolbnya thanks for confirming. I just tested this and it is working for me. Can you share what you named your branch ?

**Reply by KirillDolbnya** [reply]
in different ways, octacon, intro-to-github @queenofcorgis

**Reply by queenofcorgis** [reply]
@KirillDolbnya ah can you try with the branch name `my-first-branch`. For this course, the name must be my-first-branch to trigger the course workflow.

**Reply by KirillDolbnya** [reply]
Oh, that's what I called the branch, I thought about the repository @queenofcorgis

**Reply by queenofcorgis** [reply, marked as answer]
Ok great 😃  two options here: 

* Start the course over - head back to skills.github.com and create a new repo to see if this resolves the error
* Actions tab - in the repository where you're working on this course, there is a tab called `Actions` - if you click on that you will see something similar to the below, but if something went wrong with the Action - there will be a red icon which can tell us more about what went wrong 
![Screenshot 2024-03-27 at 11 08 23 AM](https://github.com/community/community/assets/92824518/a822d9c1-8085-4f5a-a3c6-fbafaae4fe9a)

**Reply by KirillDolbnya** [reply]
This time it worked out.
Thank you! )
@queenofcorgis

**Reply by queenofcorgis** [reply]
Amazing - great job 🎉 And you're already becoming a pro coming to the community to troubleshoot - can't wait to see what else you get up to on GitHub!
